ext/aom.cmd

```diff
@@ -14,6 +14,6 @@ cd aom
 mkdir build.libavif
 cd build.libavif
 
-cmake -G Ninja -DBUILD_SHARED_LIBS=OFF -DCMAKE_BUILD_TYPE=Release -DENABLE_DOCS=0 -DENABLE_EXAMPLES=0 -DENABLE_TESTDATA=0 -DENABLE_TESTS=0 -DENABLE_TOOLS=0 ..
+cmake -G Ninja -DBUILD_SHARED_LIBS=OFF -DCMAKE_POSITION_INDEPENDENT_CODE=ON -DCMAKE_BUILD_TYPE=Release -DENABLE_DOCS=0 -DENABLE_EXAMPLES=0 -DENABLE_TESTDATA=0 -DENABLE_TESTS=0 -DENABLE_TOOLS=0 ..
```

Comment:
In aom.cmd and avm.cmd, use `-DCONFIG_PIC=1` instead of `-DCMAKE_POSITION_INDEPENDENT_CODE=ON`.

`-DCONFIG_PIC=1` will set `CMAKE_POSITION_INDEPENDENT_CODE` to `ON` and also define the `PIC` macro for libaom/libavm's assembly code.